packages/mds-provider/utils.ts

```diff
@@ -33,6 +33,20 @@ export function asStatusChangeEvent({ event_type }: VehicleEvent | { event_type:
         event_type: VEHICLE_STATUSES.removed,
         event_type_reason: VEHICLE_EVENTS.service_end
       }
+
+    case 'rebalance_drop_off':
```

Comment:
Should move these two to the `VEHICLE_EVENTS.provider_drop_off` case ... comment below seems to indicate that's where they should go.